src/node-saml/saml.ts

```diff
@@ -468,24 +465,19 @@ class SAML {
       this.signRequest(samlMessage);
     }
     Object.keys(samlMessage).forEach((k) => {
-      target.query[k] = samlMessage[k];
+      target.searchParams.append(k, samlMessage[k] as string);
     });
 
-    // Delete 'search' to for pulling query string from 'query'
```

Comment:
I assume you've checked that the behavior here is preserved with the move to `new URL()`.

Reply:
my understanding is that that was a workaround for the deprecated URL API interface and is not necessary for the new WHATWG URL API